.circleci/config.yml

```diff
@@ -30,32 +28,28 @@ commands:
           command: git gc
       - save_cache:
           name: Git save cache
-          key: source-v1-{{ .Branch }}-{{ .Revision }}
+          key: source-v{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}
           paths:
             - ".git"
-  bundle_install:
-    description: "Bundle install"
+
+  npm_install_from_cache:
+    description: "npm install and save cache"
     steps:
       - restore_cache:
-          name: Bundle modules cache restore
-          keys:
-            - bundle-v1-{{ checksum "Gemfile" }}
-            - bundle-v1-
+          key: v1-deps-{{ checksum "package-lock.json" }}
```

Comment:
first restore will fail because this `key` doesn't exist yet. would suggest keeping cache restore disabled and enabling again first cache saving